.jenkinsci/linux-post-step.groovy

```diff
@@ -3,12 +3,12 @@ def linuxPostStep() {
     try {
       // stop write core dumps
       sh "ulimit -c 0"
+      // handling coredumps (if tests crashed)
       if (currentBuild.currentResult != "SUCCESS" && params.coredump) {
-        // handling coredumps (if tests crashed)
         def dumpsFileName = sprintf('coredumps-%1$s.bzip2',
           [GIT_COMMIT.substring(0,8)])
 
-        sh(script: "find ${$WORKSPACE} -type f -name '*.coredump' -exec tar -rvfj ${dumpsFileName} {} \\;")
+        sh(script: "find . -type f -name '*.coredump'-exec tar -cjvf ${dumpsFileName} {} \\;")
```

Comment:
This will create an archive, containing just the first file found. The rest will be left out. The working solution is to use `r` option to create an uncompressed archive. And only then it can be compressed.

Reply:
@bakhtin are you sure? I do not see any problem with file aggregation except (most likely) a typo - a missing whitespace before `-exec`.

Reply:
ow, I am wrong, there is also a `+` missing after `{}`

Reply:
that should do:
```suggestion
        sh(script: "find . -type f -name '*.coredump' -exec tar -cjvf ${dumpsFileName} {} \\+;")
```